pkg/transparentproxy/transparentproxy_experimental.go

```diff
@@ -30,6 +34,39 @@ func splitPorts(ports string) ([]uint16, error) {
 	return result, nil
 }
 
+func hasLocalIPv6() (bool, error) {
```

Comment:
this is the same as [this](https://github.com/kumahq/kuma/blob/11fc569ca57c4929fae5a0ff289b36f3d7b188e4/pkg/transparentproxy/istio/tools/istio-iptables/pkg/cmd/root.go#L165) right? do we need two copies?

Reply:
it is, but this code will be deprecated in the near future, so that's why I just copied it